matRad_calcCubes.m

```diff
@@ -46,8 +46,11 @@
 %
 
 % compute physical dose for all beams individually and together
-for i = 1:length(beamInfo)
-    resultGUI.(['physicalDose', beamInfo(i).suffix]) = reshape(full(dij.physicalDose{scenNum} * (resultGUI.w .* beamInfo(i).logIx)),dij.doseGrid.dimensions);
+scenNumNom = 1;  % calculate beam dose only for the nominal scenario
+for ixScen = scenNumNom
+   for i = 1:length(beamInfo)
+      resultGUI.(['physicalDose', beamInfo(i).suffix]) = reshape(full(dij.physicalDose{ixScen} * (resultGUI.w .* beamInfo(i).logIx)),dij.doseGrid.dimensions);
+   end
```

Comment:
I don't really understand this one.
Always computing the physicalDose for scenario 1 here instead of the passed scenario number induces problems, since e.g. for constRBE the scenario RBExD after robust optimization will all be the same.